mnefun/_mnefun.py

```diff
@@ -854,6 +895,83 @@ def run_sss_positions(fname_in, fname_out, host='kasga', opts='', port=22):
     run_subprocess(cmd)
 
 
+def run_sss_localy(p, subjects, run_indices):
+    """Run SSS locally using maxwell filter in python
+
+    For details see
+    ---------------
+    mne.preprocessing.maxwell
+    """
+    data_dir = op.join(op.dirname(__file__), 'mnefun', 'data')
+    cal_file = op.join(data_dir, 'sss_cal.dat')
+    ct_file = op.join(data_dir, 'ct_sparse.fif')
+    assert isinstance(p.tsss_dur, float) and p.tsss_dur > 0
+    st_duration = p.tsss_dur
+    assert isinstance(p.sss_regularize, string_types)
+    reg = p.sss_regularize
+
+    for si, subj in enumerate(subjects):
+        if p.disp_files:
+            print('  Maxwell filtering subject %g/%g (%s).'
+                  % (si + 1, len(subjects), subj))
+        #  locate raw and erm files
+        sss_dir = op.join(p.work_dir, subj, p.sss_dir)
+        if not op.isdir(sss_dir):
+            os.mkdir(sss_dir)
+        raw_files = get_raw_fnames(p, subj, 'raw', False, False, run_indices[si])
+        raw_files_out = get_raw_fnames(p, subj, 'sss', False, False, run_indices[si])
+        erm_files = get_raw_fnames(p, subj, 'raw', 'only')
+        erm_files_out = get_raw_fnames(p, subj, 'sss', False, False, run_indices[si])
+        prebad_file = _prebad(p, subj)
+
+        #  process raw files
+        for ii, (r, o) in enumerate(zip(raw_files, raw_files_out)):
+            if not op.isfile(r):
+                raise NameError('File not found (' + r + ')')
+            raw = Raw(r, preload=True, allow_maxshield=True)
+            raw.load_bad_channels(prebad_file, force=True)
+            raw.fix_mag_coil_types()
+            raw = filter_chpi(raw)
+            _, origin, _ = fit_sphere_to_headshape(raw.info)
+            origin /= 1000
+
+            assert isinstance(p.trans_to, string_types)
+            if p.trans_to is 'median':
+                trans_to = op.join(p.work_dir, subj, p.raw_dir, subj + '_median_pos.fif')
+                if not op.isfile(trans_to):
+                    calc_median_hp(p, subj, trans_to, run_indices[si])
+            else:
+                trans_to = p.trans_to
+
+            # estimate head position
+            # pos = _calculate_chpi_positions(raw)
+            pos = _headpos(p, subj, r, o)
+            # apply maxwell filter
+            raw_sss = maxwell_filter(raw, origin=origin, int_order=p.int_order, ext_order=p.ext_order,
+                                     calibration=cal_file, cross_talk=ct_file,
+                                     st_correlation=p.st_correlation, st_duration=st_duration,
+                                     destination=trans_to, coord_frame='head',
+                                     head_pos=pos, regularize=reg)
+
+            raw_sss.save(o, overwrite=True, buffer_size_sec=None)
+            #  process erm files if any
+            if len(erm_files) > 1:
```

Comment:
I think you mean > 0 not > 1, and you don't actually need the conditional because if `len(erm_files) == 0` then the `for` loop won't even execute